website/frontend/src/pages/CareerPage.js

```diff
@@ -1,115 +1,153 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
Updates to careers page

Reply:
Thanks @FaithDaka for these PR review comments, much appreciated and welcome! Keep it up!
Just a few comments:

1. Could we add some more detail? What are we updating on the career's page? Are we consuming the backend endpoints/service on the careers page? etc? etc?
2. Secondly, there seems to be some new conflicts that could have been brought about by the recently merged [Website SEO PR ](https://github.com/airqo-platform/AirQo-frontend/pull/761) by @Codebmk 

Thanks!